.circleci/config.yml

```diff
@@ -18,23 +43,23 @@ workflows:
           username_envar: "QUAY_USERNAME"
           password_envar: "QUAY_PASSWORD"
           requires:
-            - go-build
-          filters:
-            # Trigger the job also on git tag.
-            tags:
-              only: /^v.*/
+            - build
 
       - architect/push-to-docker:
```

Comment:
I think we need to use push-to-docker-legacy so the image tag is correct for draughtsman.

See https://github.com/giantswarm/aws-operator/blob/ca543815a0672225ca14396649ce51f9fe7502d2/.circleci/config.yml#L258